setup.py

```diff
@@ -248,6 +248,11 @@ def do_setup():
                     cmdclass=cmdclass,
                     python_requires=f'>={MIN_PYTHON[0]}.{MIN_PYTHON[1]}',
                     install_requires=REQUIREMENTS,
+                    # Explicitly declare packages so setuptools does not attempt
+                    # auto discovery.
+                    # Adapted from: https://github.com/inmanta/inmanta/pull/83
+                    # See also: https://github.com/pypa/setuptools/issues/3197
+                    packages=[],
```

Comment:
Can you try something, can you set this to

```python
import setuptools 

...
packages=find_packages()
```

Reply:
Yeah, lemme give that a shot 👀